### ISSUE TITLE
Significant iPad Performance issues

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/resistance-in-a-wire#200]
Moved from https://github.com/phetsims/resistance-in-a-wire/issues/198 per @jbphet. For https://github.com/phetsims/QA/issues/219.
iOS 12, and to a lesser extent 10, has significant performance issues. When using the profiler querry parameter I saw the frame rate on iOS 12 drop as low as 15. This occurred on the published sim as well, though less drastically. iOS 10 crashed while using profiler but did not when the querry parameter was off.  As iOS 12 is our latest OS on a newer iPad, this performance is particularly bad.  It is most noticeable while manipulating sliders, and seeing the slider lag behind the finger dragging it.  It also seemed to have moments of good performance before dropping again.

**Comment by KatieWoe** [comment]
Clarification per @ariel-phet.  Poor performance is present without profiler.  Lag felt about the same with and without it turned on.  Profiler may have made it slightly worse as there was at least one crash with it on.  Profilier was used to quantify lag.

**Comment by jbphet** [comment]
I tried the sim on several iOS devices, and the sim seems usable to me on all.  It definitely has some pauses, probably due to garbage collection.  I profiled it in the Chrome developer tools on my Windows 10 machine, and it looks like the new a11y code is adding a bit more time in each animation frame.  I don't think there is anything quick that we can do on this, so I'm going to move ahead with the next RC on the 1.6 release branch and will have others evaluate and decide if the performance is acceptable.
